### ISSUE TITLE
After updating, option --dirty stopped working.

### ISSUE THREAD

**Issue by souriscloud** [tighten/duster#112]
I had updated (in composer.json i have `^2.0`) the package and now on every call of the duster with `--dirty` option, it gives me error that to use that option, Git must exists. However I am running that inside of a docker container, which was working before update issued and which still have the git executable and also the repository is still okay, nothing changed at all, only the composer update was called, and from that time, i cannot use the --dirty switch. 

What could be the cause? I would love to believe that this is mistake of mine, but so far as I can see it doesn't seem to. 😞

**Comment by driftingly** [comment]
Hi @souriscloud, do you know the last version that was working?
Both v1 and v2 use the same underlying code from Laravel Pint.

**Comment by souriscloud** [comment]
@driftingly yo, I was out of time budget so I had to move fast and went with just linting/fixing all files, when the dirty option was not available. 😁 
Since the error started appearing after the complex update was made, I had no time to truly replicate what exactly was the cause of this problem, however its gone now.... I've tried it just now, the `--dirty` option is running again like it was before, noting that without any package changes at all! Restart of containers, flush of volumes, everything was tried before when it was not working and it was not fixed by any of these actions, however a week after its working.. I guess there could be some mismatch with terminal emulator at the time, I cant really remember if I was trying it in other terminals as well..

I'm sorry for false alarm, was mostly scanning for people with same problems to pre-catch the problem it could do on productions, so very happy that was some dev machine involving problem and not problem with package at all.
